rosbag2_compression/CMakeLists.txt

```diff
@@ -140,12 +140,14 @@ if(BUILD_TESTING)
   ament_add_gmock(test_sequential_compression_reader
     test/rosbag2_compression/test_sequential_compression_reader.cpp)
   target_include_directories(test_sequential_compression_reader PUBLIC include)
+  target_include_directories(test_sequential_compression_reader PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/../rosbag2_cpp/test")
```

Comment:
I don't think this will work. I believe we'll run into the same underlying issue that `rosbag2_cpp` is not accessible from the source tree of `rosbag2_compression`. The source debs are created from [this release branch](https://github.com/ros2-gbp/rosbag2-release/tree/release/foxy/rosbag2_compression), which does not contain the source code for `rosbag2_cpp`.

Reply:
hmm, in that case I think I'll need to just duplicate the code into `rosbag2_compression`

Reply:
Either that or install the test headers in rosbag2_cpp. Maybe it could be a separate “test” target.